tools/build-vagrant-box.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -ex
+
+WORKDIR=$(mktemp -d)
+
+function cleanup {
+    rm -rf "${WORKDIR}"
+}
+
+trap cleanup EXIT
+
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends \
+    curl \
+    unzip
+
+git clone https://github.com/jakobadam/packer-qemu-templates.git ${WORKDIR}
+
+cd ${WORKDIR}/ubuntu
+
+packer build -var-file=ubuntu1604.json ubuntu-vagrant.json
```

Comment:
I fully understand needing to use Packer to build the KVM images, but would it make sense to provide a pre-built KVM image and host it somewhere to make the process a little faster?   Maybe Github Large File Storage? This way, the shell script can `wget` the image and we have full control over the content of the image.